static/index.html

```diff
@@ -0,0 +1,11 @@
+<!doctype html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<title>Themeroller</title>
```

Comment:
That's no reason to land this already, you could just base those PRs on this branch and rebase it afterwards.